shared/templates/template_ANSIBLE_audit_rules_dac_modification

```diff
@@ -53,7 +53,7 @@
 #    
 # Inserts/replaces the rule in /etc/audit/audit.rules
 #
-- name: "Inserts/replaces the %ATTR% rule in /etc/audit/audit.rules when on x86"
```

Comment:
These were closed. What's the reason to remove them? Is it established ansible practice to avoid quotes in strings?

Reply:
Some had quotes, some didn't.  I started removing them all but then realized that someone might have a plan/standard.  I don't think it's a set practice one way or the other as far as ansible is concerned.

Reply:
I think it's fine to avoid quotes everywhere, I just wanted to figure out if that's expected. IMO we want to do what most ansible devs are used to.